app/server.rb

```diff
@@ -665,6 +665,28 @@ def getChildDepts(unit)
 end
 
 ###################################################################################################
+# Item Metrics 
+get "/api/item/metrics" do |shortArk|
```

Comment:
I agree, that makes a lot of sense. The question is how to fit in that extra API fetch. Maybe do a $.ajax call, which then fills in state?